Quick-Start-Guide/Quickstart.md

```diff
@@ -4,13 +4,13 @@ Installing GlusterFS - a Quick Start Guide
 #### Purpose of this document
 This document is intended to give you a step by step guide to setting up
 GlusterFS for the first time. For this tutorial, we will assume you are
-using Fedora 22 (or later) virtual machines (other distributions and methods can be
-found in the new user guide, below). We also do not explain the steps in
-detail here, this guide is just to help you get it up and running as
-soon as possible. After you deploy GlusterFS by following these steps,
+using Fedora 22 (or later) virtual machines.
```

Comment:
I think it's time we switched this to CentOS 7 instead. Fedora will go out of date every 6 months. At the least, let's bump this up to F26.

Reply:
I will update this to Fedora 26 for now. The Centos changes will be done as part of the larger doc revamp.